City-Bureau/city-scrapers-philly
# 🏗️ Build spider: Philadelphia Tax Review Board

## What's this PR do?
Adds a spider to scrape meetings from the website of the Philadelphia Tax Review Board. The new spider is called "phipa_trb".

## Why are we doing this?
Requested by our Philadelphia partners.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl phipa_trb -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect `test_output.csv` to ensure the data looks valid. Target website is [here](https://spokanevalley.granicus.com/ViewPublisher.php?view_id=3).

## Are there any smells or added technical debt to note? 
- This spider is also scraping a google calendar much like #14. I was somewhat tempted to create a new spider mixin for handling that scrape and this, but there are a handful of differences between these two spiders and my temptation right now is to only create a mixin if we need another Google Calendar-based spider. I'm a fan of the [rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)) to avoid premature refactoring.
